packages/agreement-process/test/agreementConsumerDocuments.test.ts

```diff
@@ -129,12 +129,6 @@ describe("agreement consumer document", () => {
   });
 
   describe("add", () => {
-    beforeEach(async () => {
```

Comment:
This usage of fakeTimers for some reason was making these tests hang during the `afterAll` I added in `utils.ts`, used to close the test browser instance for PDF generation. I did not investigate thoroughly the reason, but this fixes it

Reply:
I noticed this too. Setting the option `pipe` to `true` in the puppeteer `launch` method also solved this problem